frontend/src/components/Forms/DevPortfolioForm/DevPortfolioForm.tsx

```diff
@@ -327,4 +355,111 @@ const PRInputs = ({
   );
 };
 
+const OtherPRInputs = ({
+  otherPRs,
+  setOtherPRs,
+  openOther,
+  setOpenOther,
+  explanationText,
+  setExplanationText,
+  isTpm
+}: {
+  otherPRs: string[];
+  setOtherPRs: React.Dispatch<React.SetStateAction<string[]>>;
+  openOther: boolean;
+  setOpenOther: React.Dispatch<React.SetStateAction<boolean>>;
+  explanationText: string | undefined;
+  setExplanationText: React.Dispatch<React.SetStateAction<string | undefined>>;
+  isTpm?: boolean;
+}) => {
+  const keyDownHandler = (event: React.KeyboardEvent<HTMLInputElement>) => {
```

Comment:
I'd argue that `OtherPRInputs` doesn't apply to TPMs and shouldn't be shown at all to TPMs. Since TPMs don't have an opened/reviewed PRs requirement anyway (i.e. they can submit every dev portfolio without any reviewed/opened PRs), the TPM can just put extra PRs that don't apply under the "what did I work on" section. 

In fact, we're using the same `text` field on the DB level as the TPM one, and I see that they are currently managed as the same state on the frontend (`text, setText`), so there's a conflict there we'd have to address.

Hiding this for TPMs will save you that hassle.

Reply:
Yeah, you can remove those! Hm, I still see "Other PRs" on TPM view.
<img width="946" alt="Screenshot 2023-11-27 at 11 23 14 AM" src="https://github.com/cornell-dti/idol/assets/59291082/cabc3c12-0cb3-4624-971b-901e12ab8df3">